### PR TITLE
Remove extra resources created by Helm deployment

### PR DESCRIPTION
**Pull request by jiacheliu3**
This resolves https://github.com/Alluxio/alluxio/issues/10321

Added switched for fuse/fuse-client/jobs. The templates will be turned to empty on Helm deployment and generated for `kubectl` manual control.

### CODE REVIEW

File: integration/kubernetes/multiMaster-embeddedJournal/job/alluxio-format-journal-job.yaml.template [hunk 1]
```diff
@@ -30,7 +30,7 @@ spec:
     spec:
       containers:
         - name: alluxio-master
-          image: alluxio/alluxio:2.2.0-SNAPSHOT
+          image: alluxio/alluxio:2.3.0-SNAPSHOT
```
Comment (jiacheliu3) [inline integration/kubernetes/multiMaster-embeddedJournal/job/alluxio-format-journal-job.yaml.template]: This image version change was somehow overlooked in master branch.

**Comment by madanadit** [comment]
alluxio-bot, merge this please.

File: integration/kubernetes/multiMaster-embeddedJournal/job/alluxio-format-journal-job.yaml.template (hunk 1 above)
Comment (madanadit) [reply integration/kubernetes/multiMaster-embeddedJournal/job/alluxio-format-journal-job.yaml.template]: @ZacBlanco fyi

**Comment by madanadit** [comment]
@jiacheliu3 please manually cherry-pick to branch-2.2 and check the version strings match. thank you.

**Pull request merged**